### ISSUE TITLE
ReplyTo should be string not Array

### ISSUE THREAD

**Issue by appliquette** [nodemailer/nodemailer-sendgrid#1]
Firstly, thanks for putting this together.

When setting a reply to address I'm receiving the following error from the SendGrid API.
```
[ { message: 'Invalid type. Expected: object, given: array.',
    field: 'reply_to',
    help: 'http://sendgrid.com/docs/API_Reference/Web_API_v3/Mail/errors.html#message.reply_to' } ]
```

**Comment by dtzar** [comment]
I'm getting this as well and is pretty important to be able to set the reply_to to use this code at all.

**Comment by andris9** [comment]
Fixed in v1.0.2
